src/VisualStudio/Core/Def/ExternalAccess/VSTypeScript/Api/VSTypeScriptRemoteLanguageServiceWorkspace.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.VisualStudio.LanguageServices.ExternalAccess.VSTypeScript.Api
+{
+    /// <summary>
+    /// Used by TypeScript to acquire the RemoteLanguageServiceWorkspace.
+    /// </summary>
+    internal abstract class VSTypeScriptRemoteLanguageServiceWorkspace : Workspace
+    {
+        protected VSTypeScriptRemoteLanguageServiceWorkspace(HostServices host, string workspaceKind) : base(host, workspaceKind)
+        {
+        }
+    }
```

Comment:
```suggestion
    [Export]
    internal sealed class VSTypeScriptRemoteLanguageServiceWorkspaceAccessor
    {
        [ImportingConstructor]
        public VSTypeScriptRemoteLanguageServiceWorkspace(RemoteLanguageServiceWorkspace remoteLanguageServiceWorkspace)
        {
            RemoteLanguageServiceWorkspace = remoteLanguageServiceWorkspace;
        }
        
        public Workspace RemoteLanguageServiceWorkspace { get; }
    }
```

Reply:
@sharwell Because we can only access types in MS.VS.LS, I added the accessor interface there and just added the concrete accessor in the LiveShare assembly. Let me know if you'd like me to specify that the class is also external access.